packages/cli/src/commands/generate/config/generate_config_command.ts

```diff
@@ -99,6 +101,12 @@ export class GenerateConfigCommand
           'A path to directory where config is written. If not provided defaults to current process working directory.',
         type: 'string',
         array: false,
+      })
+      .option('version', {
```

Comment:
since --version is typically reserved for printing the program version, maybe we should call this `output-version`?

Reply:
good point, changed.